modules/claims_api/lib/bgs_service/vnp_proc_form_service.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+module ClaimsApi
+  class VnpProcFormService < ClaimsApi::LocalBGS
+    FORM_TYPE_CD = '21-22'
+
+    def vnp_proc_form_create(options)
+      vnp_proc_id = options[:vnp_proc_id]
+      options.delete(:vnp_proc_id)
+      vnp_proc_id = '<vnpProcId xis:nil=true/>' if vnp_proc_id.blank?
+      arg_strg = convert_nil_values(options)
+      body = Nokogiri::XML::DocumentFragment.parse <<~EOXML
+        <arg0>
+          <compId>
+            <vnpProcId>#{vnp_proc_id}</vnpProcId>
+            <formTypeCd>#{FORM_TYPE_CD}</formTypeCd>
+          </compId>
+        #{arg_strg}
+      EOXML
+
+      make_request(endpoint: 'VnpProcFormWebServiceBean/VnpProcFormService',
+                   action: 'vnpProcFormCreate', body:, key: 'return')
+    end
+
+    private
+
+    def convert_nil_values(options)
+      arg_strg = ''
```

Comment:
@mchristiansonVA I am thinking about moving this method into local_bgs since we are both using it.

Reply:
Hah, was just about to ask you that! Let me know if you want me to do that or just follow your lead...